### PR TITLE
increase wait for install timeout

### PR DESCRIPTION
**Pull request by mfrancisc**
Move timeout to cli args and increase the default timeout.
On crc it fails sometimes since the catalogsource becomes ready later.

```
Installing latest version of the host-operator
/Users/fmuntean/go/src/github.com/codeready-toolchain/toolchain-e2e/build/_output/bin/ksctl adm install-operator host --kubeconfig "/Users/fmuntean/.kube/config" --namespace toolchain-host-21201257 -y

Are you sure that you want to install host in namespace 'toolchain-host-21201257'
===============================
[y/n] -> response: 'y'
Namespace toolchain-host-21201257 created.
Creating CatalogSource host-operator in namespace toolchain-host-21201257.
CatalogSource host-operator created.
waiting for CatalogSource toolchain-host-21201257/host-operator to become ready
...
waiting for CatalogSource toolchain-host-21201257/host-operator to become ready
Error: failed waiting for catalog source to be ready.
```

### CODE REVIEW

File: pkg/cmd/adm/install_operator.go [hunk 1]
```diff
@@ -43,17 +44,18 @@ func NewInstallOperatorCmd() *cobra.Command {
...
 			ctx := clicontext.NewTerminalContext(term)
-			return installOperator(ctx, commandArgs, args[0], time.Second*60, cl)
+			return installOperator(ctx, commandArgs, args[0], cl)
 		},
 	}
 
 	cmd.Flags().StringVar(&commandArgs.kubeConfig, "kubeconfig", "", "Path to the kubeconfig file to use.")
 	flags.MustMarkRequired(cmd, "kubeconfig")
 	cmd.Flags().StringVar(&commandArgs.namespace, "namespace", "", "The namespace where the operator will be installed. Host and Member should be installed in separate namespaces. If the namespace is not provided the standard namespace names are used: toolchain-host|member-operator.")
+	cmd.Flags().DurationVar(&commandArgs.waitForReadyTimeout, "timeout", time.Second*180, "The max timeout used when waiting for each of the resources to be installed.")
```
Comment (alexeykazakov) [inline pkg/cmd/adm/install_operator.go:54]:
```suggestion
	cmd.Flags().DurationVar(&commandArgs.waitForReadyTimeout, "timeout", time.Second*180, "The max timeout used when waiting for each of the resources to be installed. The default timeout, if not specified, is 3m.")
```

**Review by alexeykazakov — APPROVED** [review]
Looks good.

File: pkg/cmd/adm/install_operator.go (hunk 1 above)
Comment (mfrancisc) [reply pkg/cmd/adm/install_operator.go:54]: Thanks, I can add it, but it's already showed by cobra args:
```
✗ go run cmd/ksctl/main.go adm install-operator
Error: required flag(s) "kubeconfig" not set
Usage:
  ksctl adm install-operator <host|member> --kubeconfig <path/to/kubeconfig> --namespace <namespace> [flags]

Flags:
  -h, --help                help for install-operator
      --kubeconfig string   Path to the kubeconfig file to use.
      --namespace string    The namespace where the operator will be installed. Host and Member should be installed in separate namespaces. If the namespace is not provided the standard namespace names are used: toolchain-host|member-operator.
      --timeout duration    The max timeout used when waiting for each of the resources to be installed. (default 3m0s)

Global Flags:
  -y, --assume-yes      Automatically answer yes for all questions.
      --config string   config file (default is $HOME/.ksctl.yaml)
  -v, --verbose         print extra info/debug messages

required flag(s) "kubeconfig" not set
```

Comment (alexeykazakov) [reply pkg/cmd/adm/install_operator.go:54]: Ah, ok. Than't it fine to keep it as is.

**Pull request merged**